package.json

```diff
@@ -14,7 +14,7 @@
   "private": true,
   "engines": {
     "node": "18.15.0",
-    "pnpm": "8.6.2"
+    "pnpm": "8.6.10"
```

Comment:
This is breaking the CI for some reason. Did we change it at some point? I've only updated pnpm to the latest version @vraravam

Reply:
So, whenever we upgrade the build tool, it has to be done in tandem in the main repo and the recipes repo in one shot. So, I upgrade the recipes, raise and approve the PR. Then upgrade the build tool AND the recipes in the main repo in 1 commit , and raise that PR as the next one. This doesn't fail.

Reply:
Oh ok! Sorry about that. I haven't realized we've made that change (or was that been like that ever since? 😂)... but that makes sense, so both repos have the same engines.

Nevertheless I can remove this change (once I get home) as it isn't a necessary part of this PR to work.

Generally, did this PR work well on macOS? I've only tested it on Windows so far. 😊

Thank you!

Reply:
I have removed that upgrade.